src/actions/content.js

```diff
@@ -34,6 +35,11 @@ function* loadContent(action) {
       filter: {},
     };
 
+    if (page) {
+      const { offset, limit } = page;
+      queryString.page = { offset, limit };
```

Comment:
Is there a reason we don't just do ```queryString.page = page;```?

Reply:
I saw this and was thinking the same. I assumed the reason was that if we extend our local state, we might very easily leak that information in the HTTP requests because it would be easy to forget that the whole page object is added to the URL.

Reply:
I cannot imagine of usecase to put more state into ```state.page```, but also ¯\_(ツ)_/¯